.github/workflows/release.yml

```diff
@@ -187,20 +189,22 @@ jobs:
 
           # Now upload the 'latest' versions to S3. This can be done later because the node manager
           # does not depend on these existing.
+          just package-release-assets "faucet" "latest"
+          just package-release-assets "node-launchpad" "latest"
           just package-release-assets "safe" "latest"
           just package-release-assets "safenode" "latest"
-          just package-release-assets "faucet" "latest"
           just package-release-assets "safenode_rpc_client" "latest"
           just package-release-assets "safenode-manager" "latest"
           just package-release-assets "safenodemand" "latest"
-          just package-release-assets "node-launchpad"
```

Comment:
`node-launchpad` did not have the `latest` version there, was it intentional? I added that in here.